app_data/sheets/data_list/modules.json

```diff
@@ -12,13 +12,18 @@
       "_translations": {
         "title": {
           "mx_es": true
-        }
+        },
+        "description": {}
       },
       "_translatedFields": {
         "title": {
           "eng": "After Onboarding Session"
+        },
+        "description": {
+          "eng": "onboarding"
```

Comment:
@ChrisMarsh82 For some reason whenever I sync I keep on seeing this in the json, even though it no longer exists in the sheet (as far as I can tell)...